app/validators/rules/rule_validator.py

```diff
@@ -237,6 +250,27 @@ def _validate_count_operator(self, operator):
                 value_source=first_argument,
             )
 
+    def _validate_sum_operator(self, operator):
```

Comment:
Let's discuss offline, we actually don't need to validate these operators seperately, it is covered in `new_when_rule_validator.py`. It just needs extending and porting since operator can be used outside of when rules now.

Reply:
I have updated this now as discussed 👍